.github/workflows/release.yml

```
@@ -77,9 +77,10 @@ jobs:
               'Build RPM Fedora 44',
               'Build RPM EL10',
               'Build RPM openSUSE Tumbleweed',
-              'Build APK Alpine 3.20',
-              'Build APK Alpine 3.22',
-            ];
+          'Build APK Alpine 3.20',
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)